hca-to-scea/hca2scea.py

```diff
@@ -61,15 +61,17 @@ def get_author_list(xlsx_dict):
     return author_list
 
 def generate_idf_file(work_dir, args, dataset_protocol_map, xlsx_dict, accession, idf_file_name,
```

Comment:
This file is way too long. 
It is obviously doing more than one thing and has too many reasons to change.
It needs to be broken down into more cohesive units.

Reply:
Hi @amnonkhen I'm not sure if you mean the "generate_idf_file" function which creates the idf file, or you mean the hca2scea.py file. It would be good to discuss this.

Reply:
I meant the file in general, but it is not critical to address now.
What I meant is that this files needs splitting to logically cohesive units, so it is more manageable.

Reply:
Ok, I will create a ticket for this, and address this, creating a new branch from master, once the individual pull requests for the different types of tests are merged into master.